WebGoatCore/ViewModels/StatusCodeViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
```

Comment:
Why is this reported and suppressed?

I suspect it means the Message - maybe because response.Message could be null? In that case, maybe we could make sure it's never null?